fluentd/README.md

```diff
@@ -87,6 +88,24 @@ Setup Example:
 </match>
 ```
 
+Additional parameters can be used to change the endpoint used in order to go through a proxy:
+
+* `host`: Proxy endpoint when logs are not directly forwarded to Datadog (default value is `intake.logs.datadoghq.com`)
+* `port`: Proxy port when logs are not directly forwarded to Datadog (default value is `10516`)
+* `use_ssl`: If `true`, the Agent initializes a secure connection to Datadog. In clear, TCP otherwise (default value is `true`)
```

Comment:
What does "In clear, TCP otherwise" mean? Please amplify / clarify.

Reply:
It means that it is not encrypted. Any suggestion on how to phrase it?

Reply:
```suggestion
* `use_ssl`: If `true`, the Agent initializes a secure TCP/SSL connection to Datadog. (default value is `true`)
```